packages/riverpod_lint/lib/src/lints/avoid_use_ref_inside_dispose.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/error/listener.dart';
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:riverpod_analyzer_utils/riverpod_analyzer_utils.dart';
+
+import '../riverpod_custom_lint.dart';
+
+const disposeMethod = 'dispose';
+
+class AvoidUseRefInsideDispose extends RiverpodLintRule {
```

Comment:
Could you rename the class and file name to match the lint code? avoid_ref_inside_state_dispose